### PR TITLE
📜 Landing page

### PR DESCRIPTION
**Pull request by eslamoo**
Related: https://github.com/SallaApp/Store/pull/3946

- Normal landing page
- Expired offer layout

- Not found layout

What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)

* 

What is the current behaviour? (You can also link to an open issue here)

* 

What is the new behaviour? (You can also link to the ticket here)

* 

Does this PR introduce a breaking change?

* 

Screenshots (If appropriate)

screencapture-salla-design-ar-dev-db68v6oemahrmbhb-2022-12-15-13_53_52

Screenshot 2022-12-15 at 1 52 09 PM

Screenshot 2022-12-15 at 1 54 37 PM

### CODE REVIEW

**Comment by SallaDev** [comment]
> The branch `static-landing-page` of https://github.com/SallaApp/theme-raed/pull/154 has been synced from master..✅

**Comment by SallaDev** [comment]
> The branch `static-landing-page` of https://github.com/SallaApp/theme-raed/pull/154 has Merge conflict❌

**Comment by SallaDev** [comment]
> The branch `static-landing-page` of https://github.com/SallaApp/theme-raed/pull/154 has Merge conflict❌

**Comment by SallaDev** [comment]
> The branch `static-landing-page` of https://github.com/SallaApp/theme-raed/pull/154 has been synced from master..✅

File: src/views/pages/landing-page.twig [hunk 1]
```diff
@@ -0,0 +1,100 @@
+{#
+| Variable     | Type                  | Description |
+|--------------|-----------------------|-------------|
+| page         | Page                  |             |
+| page.title   | string                |             |
+| page.content | string                |             |
+| page.url     | string                |             |
+| page.slug    | string                |             |
```
Comment (jalmatari) [inline src/views/pages/landing-page.twig:8]: add documentation for landing object, use: https://www.tablesgenerator.com/markdown_tables

File: src/views/pages/landing-page.twig [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+                    {% endif %}
+                </div>
+            </header>
+
+            <section class="s-block container overflow-hidden pb-10">
+                {% if landing.is_slider %}
+                    {% include 'components.home.products-slider' with{products:landing.products, hide_add_btn:true, show_quantity:landing.show_quantity, title:trans('pages.offer.included_products')} %}
+                {% else %}
+                    {% include 'components.home.fixed-products' with{products:landing.products, hide_add_btn:true, show_quantity:landing.show_quantity, title:trans('pages.offer.included_products')} %}
+                {% endif %}
```
Comment (jalmatari) [inline src/views/pages/landing-page.twig:71]: try to make it like this:
```
{% set hide_add_btn=true %}
{% set show_quantity=landing.show_quantity %}
{% set products=landing.products %}
{% set title=trans('pages.offer.included_products') %}
{% if landing.is_slider %}
    {% include 'components.home.products-slider' %}
{% else %}
    {% include 'components.home.fixed-products' %}
{% endif %}
```

**Review by jalmatari — CHANGES_REQUESTED** [review]

**Comment by SallaDev** [comment]
> The branch `static-landing-page` of https://github.com/SallaApp/theme-raed/pull/154 has been synced from master..✅

**Comment by SallaDev** [comment]
> The branch `static-landing-page` of https://github.com/SallaApp/theme-raed/pull/154 has been synced from master..✅

**Comment by SallaDev** [comment]
> The branch `static-landing-page` of https://github.com/SallaApp/theme-raed/pull/154 has Merge conflict❌

**Comment by SallaDev** [comment]
> The branch `static-landing-page` of https://github.com/SallaApp/theme-raed/pull/154 has Merge conflict❌

**Comment by SallaDev** [comment]
> The branch `static-landing-page` of https://github.com/SallaApp/theme-raed/pull/154 has Merge conflict❌

**Comment by SallaDev** [comment]
> The branch `static-landing-page` of https://github.com/SallaApp/theme-raed/pull/154 has Merge conflict❌

**Comment by SallaDev** [comment]
> The branch `static-landing-page` of https://github.com/SallaApp/theme-raed/pull/154 has been synced from master..✅

**Comment by SallaDev** [comment]
> The branch `static-landing-page` of https://github.com/SallaApp/theme-raed/pull/154 has Merge conflict❌

**Comment by SallaDev** [comment]
https://github.com/SallaApp/Store/pull/3946 ` is closed ❗️`

-------------------
https://github.com/SallaApp/theme-raed/pull/154 ` is been merged ✅`

-------------------

**Pull request merged**